### PR TITLE
Added homework_14_140123

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework14?

Thanks in advance!

### CODE REVIEW

File: homework_14_140123/src/MethodsImplementation.java [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+            }
+            if (Arrays.equals(arrBytes1, arrBytes3)) {
+                System.out.println("The contents of file '1' and file '3' are the same");
+            } else {
+                System.out.println("The contents of file '1' and file '3' do not match");
+            }
+        } catch (IOException e) {
+            System.out.println("File is not found");
+        } finally {
+            try {
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:400]: you don't need try block here, just close stream

File: homework_14_140123/src/MethodsImplementation.java [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+            }
+            if (line1.equals(line3)) {
+                System.out.println("The contents of file 'firstTestFile' and file 'thirdTestFile' are the same");
+            } else {
+                System.out.println("The contents of file 'firstTestFile' and file 'thirdTestFile' do not match");
+            }
+        } catch (IOException e) {
+            System.out.println("File not found");
+        } finally {
+            try {
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:359]: you don't need try block here, just close stream

File: homework_14_140123/src/MethodsImplementation.java [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+                }
+            } while (i != -1);
+            System.out.println(line);
+            bufferedWriter2 = new BufferedWriter(new FileWriter(fileCopy));
+            String newline = line.replaceAll("[aeiouAEIOU]", "");
+            bufferedWriter2.write(newline);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:298]: you don't need try block here, just close stream

File: homework_14_140123/src/MethodsImplementation.java [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+                    break;
+                } else {
+                    stringBuilder.append((char) i);
+                }
+            }
+            System.out.println(stringBuilder);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:249]: you don't need try block here, just close stream

File: homework_14_140123/src/MethodsImplementation.java [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(file));
+            bufferedWriter.write("First line\n");
+            bufferedWriter.write("Second line\n");
+            bufferedWriter.write("Third line\n");
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:185]: you don't need try block here, just close stream

File: homework_14_140123/src/MethodsImplementation.java [hunk 6]
```diff
@@ -0,0 +1,409 @@
...
+        File file = new File("TestFile.txt");
+        try {
+            file.createNewFile();
+            System.out.println(file + " is created");
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(file));
```
Comment (YeaaaaH) [inline homework_14_140123/src/MethodsImplementation.java:178]: use try with resources

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
fix repeatable issues

File: homework_14_140123/src/MethodsImplementation.java (hunk 6 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:178]: fixed. using try with resources.

File: homework_14_140123/src/MethodsImplementation.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:400]: fixed. using try with resources.

File: homework_14_140123/src/MethodsImplementation.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:359]: fixed. using try with resources.

File: homework_14_140123/src/MethodsImplementation.java (hunk 3 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:298]: fixed. using try with resources.

File: homework_14_140123/src/MethodsImplementation.java (hunk 4 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:249]: fixed. using try with resources.

File: homework_14_140123/src/MethodsImplementation.java (hunk 5 above)
Comment (kate-lvlup) [reply homework_14_140123/src/MethodsImplementation.java:185]: fixed. using try with resources.

**Pull request merged**